packages/data-explorer/public/index.html

```diff
@@ -7,6 +7,13 @@
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title>data-explorer</title>
     <script>
+    window.__INITIAL_STATE__ = {
```

Comment:
Having a static __INITIAL_STATE__ feels broken. Can we get thees values from context api?